openwisp_monitoring/monitoring/migrations/0005_create_ping.py

```diff
@@ -0,0 +1,24 @@
+from django.db import migrations
```

Comment:
this migration should go in the check app and should be executed only if `openwisp_monitoring.check.settings.AUTO_PING` is `True`.